### PR TITLE
Tweak: WP_Query - Don't calc the total rows if not needed [ED-3150]

### PR DESCRIPTION
**Pull request by iNewLegend**

### CODE REVIEW

File: core/admin/admin-notices.php [hunk 1]
```diff
@@ -53,7 +53,7 @@ private function get_install_time() {
 
 	private function get_elementor_pages_count() {
 		if ( null === $this->elementor_pages_count ) {
-			$elementor_pages = new \WP_Query( [
+			$elementor_pages = new \WP_Query( [ // 'no_found_rows' => false.
```
Comment (matipojo) [inline core/admin/admin-notices.php:56]: Go over all queries and check if needed

**Review by matipojo — CHANGES_REQUESTED** [review]

File: core/app/modules/import-export/directories/post-type.php [hunk 2]
```diff
@@ -44,7 +45,7 @@ public function export() {
 			],
 		];
 
-		$query = new \WP_Query( $query_args );
+		$query = new \WP_Query( $query_args ); // 'no_found_rows' => true.
```
[the remark this answers is not in the document]
Comment (iNewLegend) [reply core/app/modules/import-export/directories/post-type.php]: Handled.

File: core/admin/admin-notices.php (hunk 1 above)
Comment (iNewLegend) [reply core/admin/admin-notices.php]: Handled.

File: core/admin/admin-notices.php [hunk 3]
```diff
@@ -53,7 +53,7 @@ private function get_install_time() {
 
 	private function get_elementor_pages_count() {
 		if ( null === $this->elementor_pages_count ) {
-			$elementor_pages = new \WP_Query( [
+			$elementor_pages = new \WP_Query( [ // SQL_CALC_FOUND_ROWS is used.
```
Comment (matipojo) [inline core/admin/admin-notices.php:56]: Why?

File: includes/utils.php [hunk 4]
```diff
@@ -701,6 +701,6 @@ public static function get_recently_edited_posts_query( $args = [] ) {
 			'orderby' => 'modified',
 		] );
 
-		return new \WP_Query( $args );
+		return new \WP_Query( $args ); // SQL_CALC_FOUND_ROWS is used.
```
Comment (matipojo) [inline includes/utils.php:704]: Why?

**Review by matipojo — CHANGES_REQUESTED** [review]

File: core/admin/admin-notices.php (hunk 3 above)
Comment (iNewLegend) [reply core/admin/admin-notices.php]: Handled.

File: includes/utils.php (hunk 4 above)
Comment (iNewLegend) [reply includes/utils.php]: Handled.

**Review by matipojo — APPROVED** [review]

**Pull request merged**